backend/dataall/api/Objects/Environment/resolvers.py

```diff
@@ -30,16 +31,38 @@ def get_trust_account(context: Context, source, **kwargs):
     return current_account
 
 
-def check_environment(context: Context, source, input=None):
+def get_pivot_role_as_part_of_environment(context: Context, source, **kwargs):
+    ssm_param = ParameterStoreManager.get_parameter_value(region=os.getenv('AWS_REGION', 'eu-west-1'), parameter_path=f"/dataall/{os.getenv('envname', 'local')}/pivotRole/createdAsPartOfEnvironmentStack")
```

Comment:
I see in multiple places in the code returning a default region as Ireland? do we really need it?

Reply:
I will have opened another issue for this. I think it has to do with the docker deployment of data.all. But it is good that you bring it up :)

Reply:
sure thing, thanks!